frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -139,7 +139,10 @@ export default Vue.extend({
         },
     },
     computed: {
-        debtAndIncentives(): BigNumber {
+        debtAndIncentives(): BigNumber | undefined {
+            if (!this.debtDai || !this.incentiveConstantDai || !this.incentiveRelativeDai) {
+                return undefined;
+            }
```

Comment:
None of this should be possible according to the https://github.com/sidestream-tech/unified-auctions-ui/pull/454#issuecomment-1247681874